server/graphql/v1/queries.js

```diff
@@ -222,18 +237,23 @@ const queries = {
         title: get(host, 'settings.invoiceTitle') || 'Donation Receipt',
         HostCollectiveId: host.id,
         slug: args.invoiceSlug,
```

Comment:
args.invoiceSlug won't be defined when using the new arg type. Need to check the query is still coming back ok.